src/hooks/api/functions/serialize.ts

```
@@ -46,21 +54,20 @@ export async function serialize(client: Client, messages: WAMessage) {
     jid: key.remoteJid!,
     isGroup: isJidGroup(key.remoteJid!),
     owner: owner,
-    prefix,
+    prefix: prefix,
     sender: sender,
-    text: extractTextFromMessage(message!),
+    text: extractStringfromMessage(message!),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)